login.html

```diff
@@ -31,10 +31,10 @@
           <label for="nav__menu-toggle"><div class="nav__hamburger"></div></label>
           <input type="checkbox" name="nav__menu-toggle" id="nav__menu-toggle">          
           <ul>
-            <li><a href="#" class="nav__link">About Us</a></li>
+            <li><a href="login.html" class="nav__link">About Us</a></li>
             <li><a href="login.html" class="nav__link">Adopt</a></li>
             <li><a href="login.html" class="nav__link">Shelters</a></li>
-            <li><a href="#" class="nav__link">Donate</a></li>
+            <li><a href="login.html" class="nav__link">Donate</a></li>
```

Comment:
Is the donate button going to be found anywhere else? If not should we just remove it to avoid confusion?